chapters/hpmor-chapter-007.tex

```diff
@@ -452,7 +452,7 @@ \chapter{Reciprocation}
 
 It was at this point that Harry came to the sudden realisation that (a)~the sounds coming from the rest of the train platform had turned into more of a blurred white noise at around the same time Draco had reached inside his robes, and (b)~when he had discussed committing murder as a bonding method, there had been exactly one person in the conversation who’d thought they were joking.
 
-\emph{Right. Because he} seemed \emph{like such a normal kid. And he} is \emph{a normal kid, he is just what you’d} expect \emph{a baseline male child to be like if Darth Vader were his doting father.}
+\emph{Right. Because he \emph{seemed} like such a normal kid. And he} is \emph{a normal kid, he is just what you’d \emph{expect} a baseline male child to be like if Darth Vader were his doting father.}
```

Comment:
One more you missed on "is" in the first line!

Reply:
thanks, good catch!